### ISSUE TITLE
Make file compile time optimization 

### ISSUE THREAD

**Issue by Camofelix** [kiler129/relax-intel-rmrr#1]
Hey mate! lovely job on updating the source PVE thread and implementing this for 5.8+

For the section on building you own, I'd suggest changing from "make" to "make -j" to ensure that as available threads are made in parallel. essentially makes it better from the schedulers point of view, because it can better align requests and not miss the L1 or L2 cache as much for dependent threads. 

Just a suggestion! 

-FCLC

**Comment by kiler129** [comment]
Hi @Camofelix!

Thank you for the kind words, it's really appreciated!

Regarding your suggestion: I initially tried `-j32` to match my build machine. However, the Proxmox process is pretty peculiar and this does not work as expected.

The `Makefile` shipping with [`pve-kernel`](https://git.proxmox.com/?p=pve-kernel.git;a=summary) used to [crash when running in parallel](https://forum.proxmox.com/threads/compiling-proxmox-kernel-from-git.34401/#post-168802) but since then it was fixed to actually use [`dpkg-buildpackage --jobs=auto`](https://git.proxmox.com/?p=pve-kernel.git;a=blob;f=Makefile;h=7eb2715f6dc48e8ac925ff4eaf49cd81cd0917ca;hb=refs/heads/master#l58). This change causes the main Proxmox build process to run without parallelism (as there's hardly anything which can be parallel) while also building the kernel using all threads available.

So in short the heavy parts of the build are running with all threads possible, while multiple stages of the build (downloading ubuntu kernel, patching, downloading ZFS etc etc) are done synchronously.


Let me know if this clears things a bit.

**Comment by Camofelix** [comment]
> Regarding your suggestion: I initially tried `-j32` to match my build machine. However, the Proxmox process is pretty peculiar and this does not work as expected.
Gotcha! I was specifically targeting the other distros section! but If you think it doesnt add anything, the by all means! 

I'll close  the issue
